src/main/resources/application.properties

```diff
@@ -13,6 +13,9 @@ aws.cognito.jwt.jwsAlgorithm=RS256
 aws.cognito.jwt.readTimeout=2000
 aws.cognito.jwt.connectTimeout=2000
 #test.aws.cognito.idToken=<id_token>
+aws.ses.source=contact@hei.school
```

Comment:
Use environment variable instead.

So you should contact Ops or do the necessary changes in CI, CD and CloudFormation for Parameter Store.